src/App.js

```diff
@@ -3,11 +3,25 @@ import 'bootstrap/dist/css/bootstrap.min.css';
 import './App.css';
 import {Nav, Navbar, Button} from 'react-bootstrap';
 import logo from './logo.png';
-
+import Header from './Header/Header';
+import Emulator from './Emulator/Emulator';
 
 // rename App.js and App.css to navbar component
 // Move navbar.js, navbar.css, and logo(?) once file structure is determined
 
+const data = {
+  "id": 1,
+  "module_number": 1,
+  "module_name": "Local Area Networks",
+  "objective": "Connect two hosts together and check they can communicate",
+  "tasks": [
+    "Attach h1 to s1.",
+    "Attach h2 to s1.",
+    "Launch.",
+    "In Console, run h4> ping 172.16.0.100.",   
```

Comment:
`h1` ?

I'd like to do:
```
    "Attach h1 to s1.",
    "Attach h2 to s1.",
    "Launch.",
    "In Console, run h1> arp 172.16.0.100.",
    "In Console, run h1> ping 172.16.0.100.",
    "In Console, run h1> arp 172.16.0.100.",
```

with the expectation that arp will come back with empty table the first time